ReactWindows/ReactNative/Modules/Camera/CameraRollManager.cs

```diff
@@ -0,0 +1,156 @@
+using Newtonsoft.Json.Linq;
+using ReactNative.Bridge;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Windows.Storage;
+using Windows.Storage.Search;
+using Windows.Storage.FileProperties;
+
+namespace ReactNative.Modules.Camera
+{
+    /// <summary>
+    /// Native module CameraRoll.
+    /// CameraRoll provides access to the local camera roll / gallery.
+    /// </summary>
+    public class CameraRollManager : ReactContextNativeModuleBase
+    {
+        // Supported media file types
+        private static readonly List<string> SupportedVideoFileTypes = new List<string> { ".mp4", ".wmv", ".avi" };
```

Comment:
>List [](start = 32, length = 4)

Can make this `IReadOnlyList` to guarantee no one modifies the list itself (`readonly` only guarantees no one modifies the reference to `List`).